src/mixins/legal-api-mixin.ts

```diff
@@ -23,6 +24,8 @@ export default class LegalApiMixin extends Vue {
    * @returns a promise to return the filing of the specified type
    */
   async fetchFilingById (id: number): Promise<any> {
+    // Override api version for /filings requests as V2 is currently broken for Correction Filings
+    axios.defaults.baseURL = 'https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v1'
```

Comment:
Is this only temporary?

If short-term, is there a ticket to clean this up?

If long-term then I'd like to see a different approach.

Reply:
The api is(was) broken on V2 for fetching these filings, will create a ticket.

Reply:
To be fixed later. Resolving.